spark-bigtable-core/src/test/scala/com/google/cloud/spark/bigtable/ReadRowConversionsTest.scala

```diff
@@ -220,6 +220,131 @@ class ReadRowConversionsTest
     }
   }
 
+  test("buildRowWithRegexQualifier") {
+    val catalog =
+      s"""{
+         |"table":{"name":"tableName"},
+         |"rowkey":"stringCol",
+         |"columns":{
+         |"stringCol":{"cf":"rowkey", "col":"stringCol", "type":"string"}
+         |},
+         |"regexColumns": {
+         |  "s_col1": {
+         |    "cf": "cf1",
+         |    "pattern": "(?i)attr.*",
```

Comment:
I'm wondering if this test is too similar to the test data that customer sent us. Maybe we can make it more generic?